automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookBioconductorKernelSpec.scala

```diff
@@ -40,6 +40,7 @@ class NotebookBioconductorKernelSpec extends ClusterFixtureSpec with NotebookTes
         withNewNotebook(clusterFixture.cluster, RKernel) { notebookPage =>
           // Make sure BiocManager has the correct version of Bioconductor
           notebookPage.executeCell("""BiocManager::version() == "3.9"""")
+          notebookPage.saveAndCheckpoint()
```

Comment:
In your [other flaky test fix PR](https://github.com/DataBiosphere/leonardo/pull/1158/files), you also added the line `shutdownKernel()` line. Should we do that here as well?

Reply:
I don't think `shutdownKernel` is relevant...I don't want to blindly add `shutdownKernel ` everywhere. I had it in the other test because I saw improvement after adding `shutdownKernel`, and that test was a bit more complicated